optaweb-employee-rostering-frontend/src/ui/pages/shift/ExportScheduleModal.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Modal, Button, ButtonVariant, Form, InputGroup, Label } from '@patternfly/react-core';
+import DatePicker from 'react-datepicker';
+import { useTranslation } from 'react-i18next';
+import MultiTypeaheadSelectInput from 'ui/components/MultiTypeaheadSelectInput';
+import { Spot } from 'domain/Spot';
+import { AppState } from 'store/types';
+import { spotSelectors } from 'store/spot';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+interface StateProps {
+    tenantId: number;
```

Comment:
This file is breaking many code style rules. Please follow the ESLint rules that we have configured to ensure consistent code style.